tests/http_test.go

```diff
@@ -22,17 +23,21 @@ func TestWithHTTPDriver(t *testing.T) {
 	database, err := db.SetupAndMigratePostgresDB(config.DbDSN(), config.DbMaxConnections())
 	require.NoError(t, err, "setting up db failed")
 
-	err = db.EmptyTables(database.Goqu, "purls", "domains")
+	err = db.EmptyTables(database.Goqu, "purls", "domains", "user_keys", "users")
 	require.NoError(t, err, "truncating tables failed")
 
 	service := app.NewService(database.Gorm)
 	userService := app.NewUserService(database.Gorm)
+
+	key := dsl.GivenSomeUser(t, userService)
```

Comment:
I am unsure if using `dsl.GivenSomeUser` is 1) properly implemented (in the context of bdd testing) and 2) usage like this is appropriate.

Reply:
I'll have to see how authentication will fit in with these tests. Authorization is for sure something that needs testing, which will require more work.

Reply:
#90